cppcheck.spec

```diff
@@ -0,0 +1,22 @@
+### RPM external cppcheck 1.89
+Source: https://github.com/danmar/cppcheck/archive/%{realversion}.tar.gz
+
+BuildRequires: cmake gmake
+Requires: pcre
+
+%prep
+%setup -n %{n}-%{realversion}
+ 
+%build
+cmake . \
+  -DCMAKE_INSTALL_PREFIX:PATH="%{i}" \
```

Comment:
@tulamor , please use -DCMAKE_BUILD_TYPE=Release to build release mode (default is debug).